websites/A/Animeflix/presence.ts

```diff
@@ -0,0 +1,29 @@
+const presence = new Presence({
+		clientId: "1039178085922250873",
+	}),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	});
```

Comment:
You need to get strings like in this example https://github.com/PreMiD/Presences/blob/5e5f99668ba4800524e66c2a3fc9fd38755025c6/websites/S/Skribbl.io/presence.ts#L14